src/stylesheets/components/_subnav.scss

```diff
@@ -16,11 +16,10 @@
 
   .app-subnav__link {
     padding: 2px 0;
-    text-decoration: none;
+    @include govuk-link-style-no-underline;
 
     &:not(:focus):hover {
       color: $govuk-link-colour;
```

Comment:
Same comment as above about not changing link colour on hover

Reply:
@christopherthomasdesign Same question here regarding the subnav.